modules/social_features/social_profile/social_profile.install

```diff
@@ -433,3 +433,14 @@ function social_profile_update_8803() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Updates profile configuration.
+ */
+function social_profile_update_8804(&$sandbox) {
+  $config = \Drupal::configFactory()->getEditable('social_profile.settings');
+  $config->set('enable_profile_tagging', TRUE);
```

Comment:
@dmytriikaiun To keep backward compatibility I think we need to set FALSE by default for all those settings.

Reply:
@ribel I guess it better to remove hook_update